### ISSUE TITLE
train and test on PhotoTourism dataset

### ISSUE THREAD

**Issue by atoshpulatov** [zju3dv/LoFTR#235]
how can i test and train on PhotoTourism dataset

**Comment by trand2k** [comment]
> > 
> 
> Excuse me, do you use your own dataset to train the LoFTR model? Now I want to train this model with my own dataset

You have any tip? I wat train this model for my custom dataset, can you help me

**Comment by 1135451439** [comment]
> can you help me

 can you help me？

**Comment by trand2k** [comment]
> 

u need to start training LOFTR on Scannet dataset.

**Comment by trand2k** [comment]
> > can you help me
> 
> can you help me？

u need to start training LOFTR on Scannet dataset

**Comment by 1135451439** [comment]
> > > can you help me
> > 
> > 
> > can you help me？
> 
> u need to start training LOFTR on Scannet dataset

Can I train on my own data set?

**Comment by trand2k** [comment]
> > > > can you help me
> > > 
> > > 
> > > can you help me？
> > 
> > 
> > u need to start training LOFTR on Scannet dataset
> 
> Can I train on my own data set?

Yes, can i contact you in social media?. I'm trying to create dataset for training LoFTR now. But have some challenge. I can share my knowledge for you.

**Comment by 1135451439** [comment]
> > > > > can you help me
> > > > 
> > > > 
> > > > can you help me？
> > > 
> > > 
> > > u need to start training LOFTR on Scannet dataset
> > 
> > 
> > Can I train on my own data set?
> 
> Yes, can i contact you in social media?. I'm trying to create dataset for training LoFTR now. But have some challenge. I can share my knowledge for you.

Thank you very much.my email address is 1135451439@qq.com.If you have other contact information, I can also add you.

**Comment by nonathecoda** [comment]
@trand2k would you be available to chat about training loftr? I'd be happy to talk to someone who already managed, as I'm currently struggling with it!

Thanks:)

**Comment by trand2k** [comment]
> @trand2k would you be available to chat about training loftr? I'd be happy to talk to someone who already managed, as I'm currently struggling with it!
> 
> Thanks:)

yep. you can chat with me in facebook https://www.facebook.com/profile.php?id=100038969042757. or email dinhtra2000@gmail.com

**Comment by zym-njust** [comment]
Excuse me, can I ask you about making custom dataset? 
How can I get the overlap score and cental matches for the pair images? I would be very appreciate if you rely!

**Comment by arthurdu98** [comment]
HI，there. I have the same question with you on making custom dataset of LoFTR. Do you have some ideas about it? Can you share with me?  I would be very appreciate if you rely!

**Comment by trand2k** [comment]
> HI，there. I have the same question with you on making custom dataset of LoFTR. Do you have some ideas about it? Can you share with me? I would be very appreciate if you rely!

Hi arthurdu98,
You can see my discussion in this issue : https://github.com/zju3dv/LoFTR/issues/254#issuecomment-1767896858
Hope this can help you,
Good luck,
TraDinh
